command/sub8_missions/missions/align_channel.py

```diff
@@ -10,45 +10,56 @@
 
 import numpy as np
 
-marker_found = False
-
 
 @util.cancellableInlineCallbacks
 def run(sub_singleton):
+    global marker_found
+
+    print "Searching"
     nh = yield txros.NodeHandle.from_argv('marker_alignment')
-    next_pose = nh.get_service_client('/next_search_pose', SearchPose)
+    next_pose = yield nh.get_service_client('/next_search_pose', SearchPose)
     cam = PinholeCameraModel()
 
     # Go to max height to see as much as we can.
     yield sub_singleton.move.depth(0.6).zero_roll_and_pitch().go()
-    rospy.sleep(1)
+    yield nh.sleep(1.0)
```

Comment:
We must need to fix the time bug in C3, it's a shame that it's the only obstacle to doing 10x simulation.

Thompson!!!